composer.json

```diff
@@ -7,9 +7,8 @@
     "prefer-stable": true,
     "require": {
         "roave/security-advisories": "dev-master",
-        "composer/installers": "^1.0",
-        "drupal-composer/drupal-scaffold": "^2.0.0",
-        "cweagans/composer-patches": "^1.0",
+        "composer/installers": "^1.9",
+        "cweagans/composer-patches": "^1.6",
```

Comment:
@navneet0693 These two lines aren't needed either (they've been removed in `social_template`). These are still present in goalgorilla/open_social's composer.json

Reply:
@Kingdutch thanks, I will remove them.